src/components/scenes/ChangePasswordScene.tsx

```diff
@@ -182,10 +196,11 @@ const getStyles = cacheStyles((theme: Theme) => ({
 export const ChangePasswordScene = () => {
   const dispatch = useDispatch()
   const account = useSelector(state => state.account ?? undefined)
-
+  const [spinning, setSpinning] = React.useState(false)
```

Comment:
This is a good solution, but it only applies to the change-password scene, not the other 2 versions of this scene. Instead, please put the state inside `ChangePasswordSceneComponent` itself, and then put the `setSpinning` calls before and after the calls to `onSubmit` inside `handlePress` (with try / finally).